versions.tf

```
@@ -2,7 +2,7 @@ terraform {
   required_version = ">= 0.12.0, < 0.14.0"
 
   required_providers {
-    aws      = "~> 2.0"
+    aws      = "~> 3.0"
```

```suggestion
    aws      = ">= 2.0"
```
for consistency with others modules